vocab/context.jsonld

```diff
@@ -0,0 +1,87 @@
+{
+  "@context": {
+    "id": "@id",
+    "type": "@type",
+    "cred": "https://w3id.org/credentials#",
+    "dc": "http://purl.org/dc/terms/",
+    "schema": "http://schema.org/",
+    "sec": "https://w3id.org/security#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "CryptographicKey": "sec:Key",
+    "GraphSignature2012": "sec:GraphSignature2012",
+    "Identity": "https://w3id.org/identity#Identity",
+    "LinkedDataSignature2015": "sec:LinkedDataSignature2015",
+    "created": {
+      "@id": "dc:created",
+      "@type": "xsd:dateTime"
+    },
+    "creator": {
+      "@id": "dc:creator",
+      "@type": "@id"
+    },
+    "domain": "sec:domain",
+    "expires": {
+      "@id": "sec:expiration",
+      "@type": "xsd:dateTime"
+    },
+    "nonce": "sec:nonce",
+    "normalizationAlgorithm": "sec:normalizationAlgorithm",
+    "owner": {
+      "@id": "sec:owner",
+      "@type": "@id"
+    },
+    "privateKey": {
+      "@id": "sec:privateKey",
+      "@type": "@id"
+    },
+    "privateKeyPem": "sec:privateKeyPem",
+    "publicKey": {
+      "@id": "sec:publicKey",
+      "@type": "@id"
+    },
+    "publicKeyPem": "sec:publicKeyPem",
+    "publicKeyService": {
+      "@id": "sec:publicKeyService",
+      "@type": "@id"
+    },
+    "revoked": {
+      "@id": "sec:revoked",
+      "@type": "xsd:dateTime"
+    },
+    "signature": "sec:signature",
+    "signatureAlgorithm": "sec:signatureAlgorithm",
+    "signatureValue": "sec:signatureValue",
+    "Credential": "cred:Credential",
+    "CryptographicKeyCredential": "cred:CryptographicKeyCredential",
+    "Profile": "cred:Profile",
+    "RevocationList2017": "cred:RevocationList2017",
+    "claim": {
+      "@id": "cred:claim",
+      "@container": "@graph"
+    },
+    "credential": {
+      "@id": "cred:credential",
+      "@type": "@id"
```

Comment:
The `credential` field should also be: `"@container": "@graph"`. Again, should I file an issue for these sorts of issues separately or should we be making the changes now in this PR?

Reply:
Let's make the change in this PR... these are "small" changes.

Reply:
👍